settings.js

```diff
@@ -257,4 +257,22 @@ module.exports = {
             enabled: false
         }
     },
+    // http://expressjs.com/en/api.html#app.settings.table
+    //editorOptions: {
```

Comment:
Small proposed change to the option name:
```suggestion
    //httpServerOptions: {
```